meetings/breakout_meetings/2019-01-14-project-progression.md

```diff
@@ -2,7 +2,7 @@
 
 ## Links
 
-* **Ref:** https://github.com/nodejs/bootstrap/blob/master/proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md
+* **Ref:** (broken link removed)
```

Comment:
https://github.com/openjs-foundation/cross-project-council/blob/858348eff4306e2b9c49e2b9d0d26d64f493a5fd/proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md is the first version of this file; there's probably a later commit that could be linked to if desired